src/tmx_json.c

```diff
@@ -205,39 +207,164 @@ static int pjson_tile(json_t *tile_el, tmx_tile **tile_headaddr) {
 	return 1;
 }
 
+
+/* This is a rather slow solution - anyone with more spare time, feel free to rework the whole structure of
+   the tiles list to make this faster */

```

Comment:
Sounds like a good idea. Just need to add all new tiles to the hash table. Could probably do this in alloc_tile itself. I can add this to the coder later if I find the time and don't forget this (I'll try to get to it this evening).